src/main/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/KinesisClientLibConfiguration.java

```diff
@@ -166,6 +166,12 @@
      * during incremental deployments of an application).
      */
     public static final boolean DEFAULT_SKIP_SHARD_SYNC_AT_STARTUP_IF_LEASES_EXIST = false;
+    
+    /**
+     * Opt in for periodic shard sync by setting this flag to true. However, setting this to true doesn't guarantee periodic shard sync. The decision
+     * to trigger shard syncs is taken by the ShardSyncStrategyDecider
+     */
+    public static final boolean DEFAULT_ENABLE_PERIODIC_SHARD_SYNC = false;
```

Comment:
Let's not make this true/false config. Instead, let the customers choose the algorithm with constants. We can introduce another periodic shardsync alternative, that's based off of leader-election table for example. For that one we'd still be using the same config, with another value.